web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -95,51 +96,54 @@ export function ActionPicker(props: { input: ReactElement }) {
     [changeActivePicker, closeAndResetInput, resetInput]
   );
 
-  // If the input is empty, we don't want to say "No matching results found" if the user is yet to
-  // type anything. This can happen e.g. after selecting two filters.
+  const excludedClustersCopy = getExcludedClustersCopy(
+    clustersService.getRootClusters()
+  );
   const NoResultsComponent =
     inputValue.length > 0 ? (
       <EmptyListCopy>
-        <Text>No matching results found.</Text>
+        <Text typography="subtitle1" color="text.primary">
+          No matching results found.
+        </Text>
+        {excludedClustersCopy && (
+          <Text
+            color="text.primary"
+            css={`
+              display: flex;
+              align-items: baseline;
+            `}
+          >
+            <icons.Info mr={1} />
+            {excludedClustersCopy}
+          </Text>
+        )}
+      </EmptyListCopy>
```

Comment:
I feel like it should match the look of a regular search result, the small icon looks kinda off here.

Reply:
What do you think about this more "search result" look?
<img width="1214" alt="image" src="https://user-images.githubusercontent.com/20583051/230453846-ea9a985f-62cf-43ca-8d79-90466701bb49.png">

>Also it'd be nice to include this is the storybook somehow so that we don't miss this edge case when refactoring the look of the results.

Added, but I'm not sure about the approach I have taken. Initially, I added a story in `SearchBar.story.tsx` where I wrapped the entire `SearchBar` in the mocked contexts (`AppContext` and `SearchBarContext`). I then set the values in a way that caused the component to render the way I wanted.  However, I thought that it is an overkill for such simple case and I replaced it with a story similar to the other stories in `ActionPicker.story.tsx` (so I just exported the `NoResults` component and then wrapped it in `ResultsList` and then in `PickerContainer`). But now I feel that this approach doesn't scale well 😏
I will think about it tomorrow :)

Reply:
> What do you think about this more "search result" look?

That's better, though what if it used the `<Item>` component? Then the first line, "No matching results found", would match the look of other action picker items such as "Connect over SSH to…". The info about clusters could be added as another row in that item, similar to how labels are on a separate row.

If we did it this way, then perhaps this item could be included with all the other ones in Action Picker's story.

That `<EmptyListCopy>` is not set in stone, I added it when there was still this three item list if the input was empty, we can get rid of it.

Reply:
Done, I removed the `EmptyListCopy`.